src/components/Editor/index.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import { Editor } from "react-draft-wysiwyg";
+import { Container } from './styles';
+import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
+
+const TextEditor = () => {
+    const [editorState, setEditorState] = useState("");
+    const handleEdit = (val) => {
+        setEditorState(val);
+    }
+    return(
+        <Container>
```

Comment:
if you see closely you are creating an extra dom element just to add the style of a container, can we apply that to Editor itself? if yes we will remove container from DOM